apps/configurator/lib/configurator_web/controllers/game_mode_html/index.html.heex

```diff
@@ -9,6 +9,12 @@
 
 <.table id="game_modes" rows={@game_modes} row_click={&JS.navigate(~p"/game_modes/#{&1}")}>
   <:col :let={game_mode} label="Name"><%= game_mode.name %></:col>
+  <:col :let={game_mode} label="Description">
```

Comment:
Not mandatory but maybe we should move the modal to a place outside of the ```
<.table id="game_modes" rows={@game_modes} row_click={&JS.navigate(~p"/game_modes/#{&1}")}>
```
Since if you click anywhere when you open the modal you get redirected to the show of the game mode

Reply:
Well, that's something I don't like about the rowclick 😢 . But I'll take it into account